### ISSUE TITLE
NVDA feedback hub

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#7745]
Steps to reproduce:
1. German user who does not speak any other languages
2. Would like to express his ideas / to upload his issues to the NVDA community

Expected behavior:
1. A new dialog should be created in NVDA (Feedback dialog) which can be accessed in the menu or by pressing a certain key stroke. The dialog contains a combo box where the user can choose his language and a second one where he can choose if it is a issue or a pull request. Then he tabs into a bunch of formfields where he can type in his issue / pull request. The language of the interface could be the OS language.
2. A data base with the translating maintainers of each country (Name, E-Mail address) should be created
3. The data base should be linked to the languages of the combo box in the feedback dialog
4. If the user chose german and submited his Feedback, the maintainers of the german community become an e-mail. At the same time, the feedback is posted in their github, send desk profile or what so ever. If someone of the maintainers feel confortable to solve the issue and to share it with the whole international community, he can do so by checking the option (in process by) followed by the name of the maintainer. If no one of the maintainers is able to solve it, then they can translate it and share it with the whole international community by checking an option which says "open for public discussion".

Of course this does not imply that the current method of giving feedback should disappear. But the new method would allow people with limited access to more than one language to give their feedback as well.

### Actual behavior:
Giving feedback is quite hard for a beginner and is linked with a registration. The process is not very intuitive and it does not allow people with limited language skills to give their feedback.

### System configuration:
NVDA version:
all

NVDA Installed or portable:
both

Other information:

Windows version:
all

Name and version of other software in use when reproducing the issue:
n/a

Other questions:

Does the issue still occur after restarting your PC?
yes

Have you tried any other versions of NVDA?
all

**Comment by josephsl** [comment]
Hi, this is what user lists are for: we do have folks over there who can translate messages and post feedback on their behalf. Also, note that not all feedback we get are strictly NVDA or accessibility related. But because this may spur follow-up discussions (including a second round of talks at NVDACon later), I suggest keeping this open. Thanks.

**Comment by fernando-jose-silva** [comment]
I see that some discussion lists like the one in Portuguese Brazil are almost inactive.
Very few e-mails come to her, and when she asks herself something in it the answer takes a long time to come, this when it comes.
The attention gained when opening a call in github is much greater.
I used to talk to you, I used the google translator, and although it looks awful you always supported me.
My suggestion would be to guide people to take advantage of these translation features to open their questions in github, or even to open calls in a custom interface of nvda.

**Comment by feerrenrut** [comment]
While I certainly agree with the intent of this, I dont think this is something that will be addressed through NVDA. I'm going to close this issue as `wontfix`.
